client/src/app/components/managing-body-list/managing-body-list.component.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Created by amanecer on 16/05/2016.
+ */
+import {Component,Input,Output,EventEmitter} from '@angular/core';
+import {ManagingBodyComponent} from './managing-body/managing-body.component';
```

Comment:
@amanecer2 Add one whitespace character inside the curly braces in the import (at the start and at the end) [Style 03-05]